include/EVT/dev/LCD.hpp

```diff
@@ -74,22 +77,86 @@ class LCD {
      */
     void displayMap(const uint8_t* bitMap);
 
+    /**
+     * Displays the given bit map at a certain height and width on the page.
+     * @param bitMap the bitmap to display.
+     * @param bitMapWidth the width of the bitmap in pixels.
+     * @param bitMapHeight the height of the bitmap in pixels.
+     * @param page the page to draw the bitmap on. Range: 0-7.
+     * @param column the column to draw the bitmap on. Range:0-127.
+     */
+    void displayBitMap(uint8_t* bitMap, uint8_t bitMapWidth, uint8_t bitMapHeight, uint8_t page, uint8_t column);
+
+    /**
+     * Writes text to the screen. Has options to wrap the text around the edge of the screen if needed.
+     * @param text the text to write to the screen.
+     * @param page the page to write the text too. Range 0-7.
+     * @param column the column to write the text too. Range 0-127.
+     * @param wrapText whether or not the text should be wrapped around the edge of the screen.
+     */
+    void writeText(const char* text, uint8_t page, uint8_t column, bool wrapText);
+
+    /**
+     * Set the default section titles to be displayed.
+     * @param newSectionTitles an array of section titles to display.
+     */
+    void setDefaultSections(char* newSectionTitles[9]);
+
+    /**
+     * Displays the section headers. Only needs to be called once unless cleared.
+     */
+    void displaySectionHeaders();
+
+    /**
+     * Set the text for a certain section of the screen.
+     * @param section the section number to set the text for.
+     * @param text the text to write into the section.
+     */
+    void setTextForSection(uint8_t section, const char* text);
+
     /**
      * Initializes the LCD for operation (must be called to use the LCD)
      */
     void initLCD();
 
 private:
-    /** register select pin for the LCD */
+    /** The total width of the screen */
+    static const uint8_t screenSizeX = 128;
```

Comment:
Likewise, we should try to abstract some of these values away, so the LCD class can be generic for use with other LCDs. It's not a priority right now, but it's something to think about.